src/generics/utils.js

```diff
@@ -936,7 +972,9 @@ function getDynamicFilterCondition(filters, columnMappings, baseQuery, columnCon
 		console.log('Filters is not an object or is empty')
 		return '' // Early exit if filters are not valid
 	}
-
+	if (filters.session_type && filters.session_type.includes('ALL')) {
+		filters.session_type = ['PUBLIC', 'PRIVATE']
```

Comment:
@sumanvpacewisdom can not we add default values for ALL in config its self ?

Reply:
this is not generic cod, if one more session type added, we need to modify the code